fdbbackup/backup.actor.cpp

```diff
@@ -3298,6 +3299,157 @@ Optional<Database> connectToCluster(std::string const& clusterFile,
 	return db;
 };
 
+class ParsedArgs {
+public:
+	ParsedArgs() = default;
+	ParsedArgs(const ParsedArgs&) = delete;
+	ParsedArgs& operator=(const ParsedArgs&) = delete;
+	ParsedArgs(ParsedArgs&&) = delete;
+	ParsedArgs& operator=(ParsedArgs&&) = delete;
+
+	void buildArgs(int& argc, const char**& argv) const {
+		argc = static_cast<int>(argvStorage.size() - 1); // exclude nullptr termination
```

Comment:
Do not use null-termination

```suggestion
		argc = static_cast<int>(argvStorage.size());
```